src/app/components/tabs.tsx

```diff
@@ -60,13 +59,13 @@ export function Tabs({
 }) {
   return (
     <Stack
-      position="relative"
-      isInline
-      borderRadius="12px"
       bg={color('bg-4')}
-      p="extra-tight"
-      width="100%"
+      borderRadius="10px"
       flexShrink={0}
```

Comment:
Is this needed?

Reply:
Yes, it rounds the corners.

Reply:
I was talking about the `flexShrink={0}`. It's not problem, it just seems unnecessary

Reply:
Haha, no I don't think this is needed so I'll remove it in the new redo PR.